web_poet/page_inputs/client.py

```diff
@@ -72,12 +73,12 @@ def _handle_status(
             return
 
         status = HTTPStatus(response.status)
-        msg = f"{response.status} {status.name} response for {response.url}"
+        msg = f"{response.status} {status.name} response for {response.url!s}"
```

Comment:
I don’t think this change makes a difference, I believe f-strings cast their parameters to string by default.

> The only reason to use '!s' is if you want to specify a format specifier that applies to str, not to the type of the expression. — [PEP 498 – Literal String Interpolation](https://peps.python.org/pep-0498/)

Reply:
Nice catch!